Syntax highlighting for curly brackets on heex

nettinho opened the pull request:
I've forked this repo to get syntax highlight working with the newly added curly brackets syntax.
I'm not sure if this is the way to submit a possible contribution, and it definitely needs revision.

SteffenDE commented:
Do you think it would be possible to respect `phx-no-curly-interpolation` and not treat curly interpolation inside tags with that attribute as Elixir expression?

Also, is there a specific reason to create a new elixir-curly-embedded definition? From my testing it looks the same when I just add 
```
{
  "include": "#interpolation"
}
```
to the root patterns. And I don't think we need the special handling for a comment.

SteffenDE commented:
I experimented with `phx-no-curly-interpolation` in https://github.com/phoenixframework/vscode-phoenix/pull/30, but I don't think it's possible.

msaraiva commented:
@SteffenDE I believe you're correct. Adding:

```
{
  "include": "#interpolation"
}
```

to the root `patterns` node should be enough. That's how we've been using it in Surface. I might have missed some edge cases, but if they exist, they just haven't been reported yet.

nettinho commented:
Strangely, this was the first thing I tried, but for some reason it didn't work for me (probably because of something else). So I moved on.
Trying it again, it did work.
I've updated the merge request, just to add `#interpolation` to the root patterns.

SteffenDE commented:
LGTM! :)

nettinho commented:
Is there no need to update the version or generate a new `.vsix` file?

msaraiva commented:
It depends on how it's going to be published. If using, for instance, `vsce publish 0.1.3 -m "Bump version to v%s"`, it already updates the version in `package.json`, creates a commit/tag for the release and publishes it. Otherwise, each one of those steps has to be done manually.

msaraiva commented:
@nettinho BTW, thank you for the PR! 💜
